lib/response-parsers.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const { InvalidResponse } = require('../services/errors')
+const xml2js = require('xml2js')
```

Comment:
With `fast-xml-parser`, the parser is now used in two places, so keeping the `require` up there. 😄

Reply:
Ah, sorry, I meant the functions themselves, not the `require`s. Instead of having these in a separate module, could they just be inlined into the `_requestX` methods in the base services?